docs/courses/design/before-you-begin.md

```diff
@@ -0,0 +1,21 @@
+# Before you begin
+
+In order to get started, you need some basic skills in writing markdown and using GitHub, and an idea of the type of course you will make.
+
+## Learn to write markdown
+
+This document is written in Github-flavored markdown. If you don't know how to write it, please take [this tutorial](https://www.markdowntutorial.com) or read [this cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+
+## Learn to edit files with github
+
+The course spec is written entirely in the `README.md` file. GitHub has instructions on [how to edit a file](https://help.github.com/articles/editing-files-in-your-repository). In that document, "add your commit" means "save your changes to the file".
```

Comment:
I would only suggest adding that "add your commit" is in step 6. It's the first thing I looked for when I opened that doc and it didn't jump out.

Reply:
fixed